init.lua

```diff
@@ -501,6 +501,11 @@ require('lazy').setup({
           --  For example, in C this would take you to the header.
           map('gD', vim.lsp.buf.declaration, '[G]oto [D]eclaration')
 
+          -- Formats the current buffer
+          map('<space>f', function()
```

Comment:
Should this be `<leader>` instead of `<space>`? It might be the same at the moment, but if someone changes leader it might be confusing as to why formatting "stopped working"

Reply:
Good point :). Updated 👍🏼